jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListViewImpl.java

```diff
@@ -41,5 +68,48 @@ public void initDefaultFilters() {
     public String getGridGlobalPreferencesKey() {
         return DATA_SET_TASK_LIST_PREFIX;
     }
+    
+    /*@Override
+    public void initColumns(ExtendedPagedTable<TaskSummary> extendedPagedTable) {
+        super.initColumns(extendedPagedTable);
+        
+        Column<TaskSummary, ?> errorCountColumn = initErrorCountColumn();
+        extendedPagedTable.addSelectionIgnoreColumn(errorCountColumn);
+        
+        extendedPagedTable.addColumn(errorCountColumn, constants.Errors());
+        extendedPagedTable.setColumnWidth(errorCountColumn,
+                                          65,
+                                          Style.Unit.PX);
+        
+    }*/
```

Comment:
@mcivantos-tribalyte This commented out code should be removed completely, if it is superfluous.